doc/ddl/ddl_reference.tex

```diff
@@ -161,15 +161,44 @@ \subsection{The \emph{data} type}
 ]
 \end{lstlisting}
 
+\subsection{map}
+The \emph{map} type the interface makes the demo show a map of the Earth where the user can drawn one or more polygons interactively. The selection is passed to the demo's code as a list of GeoJSON features containing geometric coordinates.
+
+\begin{figure}[h]
+\centering
+\tcbox[sharp corners, boxsep=0.0mm, boxrule=0.5mm, colback=white]{\includegraphics[scale=0.25]{Images/geojson_example.png}}
+\caption{The IPOL's map interface. 1) The controls to draw and remove polygons, 2) A polygon already draw, 3) A polygon being drawn, and 4) the information about the current polygon.}
+\label{fig:geojson_example}
+\end{figure}
+
+Figure~\ref{fig:geojson_example} shows the control and its key elements.
+To draw a polygon there is a toolbox (1) which allows to start drawing a polygon and to remove completely the last one. Click on the upper icon to start drawing and click on the map to add as many vertices as needed. You can also start adding vertices by clicking the right button of the mouse. When done, click the right button of the mouse. After that, the polygon will appear as finished (2). You can drawn more than one polygon, if needed. After finishing with the first, you can draw a second one (3). The information about the current polygon is show above (4).
+
+The demo will receive a GeoJSON file containing the coordinates as a list of geometric features. Please check the code of demo 11111000001 to see an example on how to parse the GeoJSON file to extract the geometric coordinates.
```

Comment:
Can you include an example of content of file `input_0.geojson`?

Reply:
Added.